Sources/Core/ARM/SoftwareBreakpointManager.cpp

```diff
@@ -131,6 +117,8 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
   }
 #endif
 
+  opcode.clear();
```

Comment:
Nope, was just not happy with push_back() on something that might already have a value. I'll keep it in this commit then.